setup.py

```diff
@@ -26,7 +26,7 @@ def package_data(pkg, root_list):
     author="MITx",
     zip_safe=False,
     install_requires=[
-        'django>=1.8,<=1.11',
+        'django>=1.8,<=1.11.11',
```

Comment:
Maybe this should be `<2.0`? What do you think?

Reply:
Good idea